src/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -286,7 +286,8 @@ export const getWizardStepClusterStatus = (
     const allHostsReady = (cluster?.hosts || []).every((host) =>
       allowedStatuses.includes(getWizardStepHostStatus(host, wizardStepId)),
     );
-    return allHostsReady &&
+
+    return (wizardStepId === 'cluster-details' || allHostsReady) &&
```

Comment:
We should keep this function as simple as possible. All the wizard step variations and specifics should be covered inside `wizardStepsValidationsMap`. This specific problem we have - being unable to move to the next step from cluster-details step - is caused by incorrect `clusterDetailsStepValidationsMap.hosts.allowedStatuses`. No host validation failure should prevent us from moving from cluster-details step.

The easiest solution should be to add 'insufficient' to the `clusterDetailsStepValidationsMap.hosts.allowedStatuses`

Reply:
Or even easier, setting `clusterDetailsStepValidationsMap.hosts.allowedStatuses` to `[]` should have the same effect

Reply:
But the host status is `error`. Should we add that too/instead? @jtomasek

Reply:
lgtm